src/inc/regdisp.h

```diff
@@ -72,7 +71,71 @@ struct REGDISPLAY : public REGDISPLAY_BASE {
     DWORD * pEax;
 
     DWORD * pEbp;
+#endif // !WIN64EXCEPTIONS
+
+#ifndef WIN64EXCEPTIONS
+
+#define VOLATILE_REG_METHODS(reg) \
+    inline PDWORD Get##reg##Location(void) { return p##reg;  } \
+    inline void   Set##reg##Location(PDWORD p##reg) { this->p##reg = p##reg; }
+
+#define NONVOLATILE_REG_METHODS(reg) VOLATILE_REG_METHODS(reg)
+
+#else // !WIN64EXCEPTIONS
+
+#define VOLATILE_REG_METHODS(reg) \
+    inline PDWORD Get##reg##Location(void) { return &pCurrentContext->reg; } \
+    inline void   Set##reg##Location(PDWORD p##reg) { pCurrentContext->reg = *p##reg; }
```

Comment:
@janvorli Could you let me know the role of ``pCurrentContext``? It seems that ``pCurrentContextPointers`` is enough for the purpose of unwind.